app/api/migrations/migrations/34-remove-_id-from-filter-items/specs/34-remove-_id-from-filter-items.spec.js

```diff
@@ -0,0 +1,31 @@
+import testingDB from 'api/utils/testing_db';
+import migration from '../index.js';
+import fixtures from './fixtures.js';
+
+describe('migration remove-_id-from-filter-items', () => {
+  beforeEach(async () => {
+    spyOn(process.stdout, 'write');
+    await testingDB.clearAllAndLoad(fixtures);
```

Comment:
The migration looks good, and I believe it works as expected.

Still, I am concerned that the fixtures only show a single scenario: "A collection whose settings have filters, and those filters have nested items with _id present".

There are many other scenarios that need to be tested that they are not affected:
- Collections that don't have filters
- Collections that have filters and don't have nested items
- Collections that have filters with nested items and no _id present in there.

Reply:
I have added a few test cases as suggested, you can check them out when you get time